crates/re_ui/src/list_item2/other_contents.rs

```diff
@@ -0,0 +1,83 @@
+use crate::list_item2::{ContentContext, DesiredWidth, ListItemContent};
+use crate::ReUi;
+use egui::Ui;
+
+/// Empty list item content.
+pub struct EmptyContent;
```

Comment:
Will this be used ever?

Reply:
I've used it at some point, but we can remove it. Worst case a `CustomContent` with an empty closure can be used.